.travis.yml

```diff
@@ -1,6 +1,7 @@
 sudo: false
 language: php
 php:
+  - 7.0
   - 5.6
   - 5.5
   - 5.4
```

Comment:
Need to put `5.4` into the allowed failures or remove for the build/tests to succeed.
```yml
matrix:
  allow_failures:
    - php: 5.4
```